src/JoomlaBrowser.php

```diff
@@ -268,7 +268,7 @@ public function installJoomlaMultilingualSite($languages = array())
 		foreach ($languages as $language)
 		{
 			$I->debug('I mark the checkbox of the language: ' . $language);
-			$I->click(['xpath' => "//label[contains(text()[normalize-space()], '$language')]/input"]);
+			$I->click(['xpath' => "//label[contains(text()[normalize-space()], '$language')]"]);
```

Comment:
About the first one, yes, that's the appropriate path, however my intention was clicking on the `<label>` tag since that works on the browsers too, and provides backwards compatibility.  It can be changed without problem of course if you think it's best to provide the exact path.